docs/src/assets/scss/syntax-highlighter.scss

```diff
@@ -99,6 +99,39 @@ pre[class*="language-"] {
     cursor: help;
 }
 
+.token.eslint-marked {
+    /* Draw the wavy line. */
+    background:
+        url("data:image/svg+xml,%3Csvg%20xmlns%3D'http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg'%20viewBox%3D'0%200%206%203'%20enable-background%3D'new%200%200%206%203'%20height%3D'3'%20width%3D'6'%3E%3Cg%20fill%3D'%23f14c4c'%3E%3Cpolygon%20points%3D'5.5%2C0%202.5%2C3%201.1%2C3%204.1%2C0'%2F%3E%3Cpolygon%20points%3D'4%2C0%206%2C2%206%2C0.6%205.4%2C0'%2F%3E%3Cpolygon%20points%3D'0%2C2%201%2C3%202.4%2C3%200%2C0.6'%2F%3E%3C%2Fg%3E%3C%2Fsvg%3E")
+        repeat-x
+        bottom
+        left;
+
+    /*
+     * Since the character width of the token span is not constant,
+     * if we use it as is, we may see a shift in the border.
+     * To make the border shift less noticeable, draw it with a smaller width.
+     */
+    background-size: 4.4px auto;
+}
+
+.token.eslint-marked-on-zero-width {
+    position: relative;
+
+    /* Delete the wavy line. */
+    background: none;
+}
+
+.token.eslint-marked-on-zero-width::after {
```

Comment:
For newlines, marks appear on the next line:

![image](https://github.com/eslint/eslint/assets/44349756/e9b79de0-76e9-4230-bfdc-699b59e0ec69)

Maybe we should use `::before` instead of `::after`?

https://deploy-preview-18041--docs-eslint.netlify.app/rules/semi#always

Reply:
Thank you for finding the problem! I fixed it.